<?xml version="1.0"?>
<sdf version="1.6">
  <model name="part_100g">
    <link name="link">
      <pose>0 0 0 0 0 0</pose>

      <inertial>
        <mass>0.1</mass>
        <pose>0 0 0 0 0 0</pose>
        <inertia>
          <ixx>6.458e-6</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>6.458e-6</iyy>
          <iyz>0</iyz>
          <izz>1.125e-5</izz>
        </inertia>
      </inertial>

      <collision name="collision">
        <geometry>
          <cylinder>
            <radius>0.015</radius>
            <length>0.0177</length>
          </cylinder>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>0.8</mu>
              <mu2>0.8</mu2>
            </ode>
          </friction>
          <contact>
            <ode>
              <kp>1000000.0</kp>
              <kd>100.0</kd>
              <max_vel>1.0</max_vel>
              <min_depth>0.001</min_depth>
            </ode>
          </contact>
        </surface>
      </collision>

      <visual name="visual">
        <geometry>
          <cylinder>
            <radius>0.015</radius>
            <length>0.0177</length>
          </cylinder>
        </geometry>
        <material>
          <ambient>0.6 0.6 0.7 1</ambient>
          <diffuse>0.6 0.6 0.7 1</diffuse>
          <specular>0.5 0.5 0.5 1</specular>
        </material>
      </visual>
    </link>
  </model>
</sdf>
